### DISCUSSION TITLE
How to add a new pip package to the project ?

### DISCUSSION THREAD

**Discussion by nelsonmandeladev** [cookiecutter/cookiecutter-django#4084] [Q&A] [answered]
I'm having a trouble on how to add new pip package to the project as it's runing on docker. I've also posted this question on stackoverflow: https://stackoverflow.com/questions/75143030/how-to-add-a-new-pip-package-to-the-project

Please for help

**Comment by browniebroke** [comment, marked as answer]
> I'm actually copying the version of the package and pasting into the base requirements file and building the local.yaml contener again and it's seams to be rebuilding the entire containers in the project instead of building only the container that changes has been detected

You mention that in StackOverflow, and that's pretty much the workflow, yes:

1. Add dependency to the relevant requiremnts file (base.txt, local.txt or production.txt)
2. Rebuild the relevant docker images
3. Restart the containers

I don't think there is any way around it, it's how Docker works.

**Reply by nelsonmandeladev** [reply]
@browniebroke , Thank for your reply
